File: app/src/main/java/ru/samtakoy/core/services/learn_courses/UncompletedTaskSettings.java
package ru.samtakoy.core.services.learn_courses;

public class UncompletedTaskSettings {

    /**
     * задержка при смахивании
     */
    private int mShiftMillis;
    /**
     * на проверку при открытии курса
     * TODO этот параметр пока решил не использовать, во всех случаях используется первый
     */
    private int mOpenCourseShiftMillis;

    public UncompletedTaskSettings(int shiftMillis, int openCourseShiftMillis) {
        mShiftMillis = shiftMillis;
        mOpenCourseShiftMillis = openCourseShiftMillis;
    }

    public int getShiftMillis() {
        return mShiftMillis;
    }

    public int getOpenCourseShiftMillis() {
        return mOpenCourseShiftMillis;
    }
}

File: app/src/main/java/ru/samtakoy/features/notifications/learn_courses/UncompletedTaskSettings.kt
package ru.samtakoy.features.notifications.learn_courses

data class UncompletedTaskSettings(
    /**
     * задержка при смахивании
     */
    val shiftMillis: Int,
    /**
     * на проверку при открытии курса
     * TODO этот параметр пока решил не использовать, во всех случаях используется первый
     */
    val openCourseShiftMillis: Int
)
